File: BaekJoon Online Judge/C++/Step08-Math_1 Mathematical Thinking/10250.cpp
//
//  10250.cpp
//  Step08-Math_1 Mathematical Thinking
//
//  Created by Yebin Kim on 2019/10/23.
//  Copyright © 2019 김예빈. All rights reserved.
//

#include <iostream>

using namespace std;

// Time Complexity O(1)
int main() {
    cin.tie(NULL);
    ios::sync_with_stdio(false);
    
    int tc, h, w, n, floor, remain;
    int answer;
    
    cin >> tc;
    
    while(tc--) {
        cin >> h >> w >> n;
        
        n--;
        floor = (n % h + 1) * 100;
        remain = n / h + 1;
        
        answer = floor + remain;
        
        cout << answer << "\n";
    }
    
    return 0;
}

File: BaekJoon Online Judge/Swift/Step08-Math_1 Mathematical Thinking/10250.swift
//
//  10250.swift
//  Step08-Math_1 Mathematical Thinking
//
//  Created by Yebin Kim on 2019/11/15.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

let tc = Int(readLine()!)!

for _ in 0..<tc {
    let input = readLine()!.split(separator: " ").map{ Int($0) }
    let H = input[0]!
    let N = input[2]! - 1
    
    let answer = "\(N % H + 1)\(String(format: "%02d", N / H + 1))"
    
    print(answer)
}
